### ISSUE TITLE
Mozilla would like the 'new room' button to default to public rather than private

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#14779]
I don't think we can configure this as a setting currently; if we can make it a granular setting and put it into their config.json it' be appreciated.

**Comment by t3chguy** [comment]
Any reason for this to be a Granular Setting over a simple config.json flag?

Do you foresee us wanting to expose this as a toggle on the account/device level?

**Comment by Twi1ightSparkle** [comment]
Specifically what they want is:
- When creating a room:
- `Make this room public` should default to `enabled`
- and `Who can access this room?` should default to `Anyone who knows the room's link, including guests`
- and `Who can read history?` should default to `Anyone`

See also https://github.com/vector-im/riot-web/issues/11918

**Comment by turt2live** [comment]
> Any reason for this to be a Granular Setting over a simple config.json flag?

Probably to allow non-mozilla folk to default to this as well, as many people tend to want features like this. More practically for Mozilla, they'd want a .well-known config option instead.

**Comment by Twi1ightSparkle** [comment]
I imagine the best would be to add this as config options to the Element config file?

**Comment by t3chguy** [comment]
> Probably to allow non-mozilla folk to default to this as well

Anyone can tweak config.json value though?

> I imagine the best would be to add this as config options to the Element config file?

@Twi1ightSparkle but then it wouldn't apply to Element Desktop

**Comment by Twi1ightSparkle** [comment]
>  but then it wouldn't apply to Element Desktop

Good point. Idk what solution then. Using .well-known as Travis suggested might be a better idea

**Comment by turt2live** [comment]
> Anyone can tweak config.json value though?

Not the people stuck on desktop (easily enough where I'd recommend they try it) or app.element.io

**Comment by ara4n** [comment]
putting the setting into .well-known sounds okay to me.

Turns out it's not just Mozilla who's after this but other VIPs also - we should probably specify the default permissions for the 'new room' button at relatively fine granularity.

**Comment by turt2live** [comment]
Putting this into the design queue for UX around specifying what https://github.com/vector-im/riot-web/issues/14779#issuecomment-665037558 means
